README.md

```diff
@@ -254,6 +254,28 @@ yarn stats
 
 ### Settings
 
+#### Organization Configuration
+
+Organization configuration can be generated by running:
+`yarn add-org`
+
+Organization configuration is generated from the template present in `internals/generators/config.yml.hbs`.
```

Comment:
BTW, why aren't you using articles? I would change this to "The configuration of organizations is generated ...".